src/components/plots/barPlot.ts

```diff
@@ -142,6 +159,31 @@ export module Plots {
       }
     }
 
+    public addDataset(dataset: Dataset) {
+      if (this._isSetup) {
+        var labelArea = this._renderArea.append("g").classed(Bar._LABEL_AREA_CLASS, true);
```

Comment:
Is there a way to move this logic to whatever logic is normally used to create new `<g>`s for the `Drawer`s?